gensim/models/word2vec.py

```diff
@@ -342,8 +342,8 @@ class Word2Vec(utils.SaveLoad):
     """
     def __init__(
             self, sentences=None, size=100, alpha=0.025, window=5, min_count=5,
-            max_vocab_size=None, sample=0, seed=1, workers=1, min_alpha=0.0001,
-            sg=1, hs=1, negative=0, cbow_mean=0, hashfxn=hash, iter=1, null_word=0,
```

Comment:
Word2Vec's switch from `hs=1, negative=0` to `hs=0, negative=5` is a fairly meaningful and helpful one as far as I can tell, motivated by improved performance, so I think it's an important place to match them. Any particular feature that now needs 'hs=1' to be specified (such as scoring) should explicitly do that, rather than forcing a less-preferred default on the larger userbase.